tests/conftest.py

```diff
@@ -557,3 +561,52 @@ def login(profile, job_application):
             raise ValueError(f"Invalid profile: '{profile}'")
 
     return login
+
+
+## Mockers
+@pytest.fixture(name="sentry_respx_mock")
+def sentry_respx_mock_fixture(respx_mock):
+    json = {
+        "data": [
+            {
+                "tuple(('duration', 300))": [["duration", 300]],
+                "apdex()": 0.9556246545071905,
+                "failure_rate()": 0.08123341283760084,
+                "project_threshold_config": ["duration", 300],
+            }
+        ],
+        "meta": {
+            "fields": {
+                "tuple(('duration', 300))": "string",
+                "apdex()": "number",
+                "failure_rate()": "percentage",
+                "project_threshold_config": "string",
+            },
+            "units": {
+                "tuple(('duration', 300))": None,
+                "apdex()": None,
+                "failure_rate()": None,
+                "project_threshold_config": None,
+            },
+            "isMetricsData": False,
+            "isMetricsExtractedData": False,
+            "tips": {"query": None, "columns": None},
+            "datasetReason": "unchanged",
+            "dataset": "discover",
+        },
+    }
+
+    end = datetime.datetime(2024, 12, 3, 0, 0, 0, tzinfo=datetime.UTC)
+    start = end - relativedelta(days=1)
+    params = {
+        "query": '(event.type:"transaction")',
+        "project": settings.API_SENTRY_PROJECT_ID,
+        "field": ["apdex()", "failure_rate()"],
+        "start": start.isoformat(),
+        "end": end.isoformat(),
+    }
+
+    url = f"{settings.API_SENTRY_BASE_URL}/organizations/{settings.API_SENTRY_ORG_NAME}/events/"
+    return respx_mock.route(
+        method="GET", params=params, url=url, headers={"Authorization": f"Bearer {settings.API_SENTRY_STATS_TOKEN}"}
+    ).mock(return_value=httpx.Response(200, json=json))
```

Comment:
PI il y a un raccourci ;) :
```suggestion
    ).respond(json=json)
```